.travis.yml

```diff
@@ -16,7 +16,7 @@ script:
 
 env:
   global:
-    - TRAVIS_CARGO_NIGHTLY_FEATURE=chan_select
+    - TRAVIS_CARGO_NIGHTLY_FEATURE=""
```

Comment:
Can this be removed?

Reply:
No, if we don't define `TRAVIS_CARGO_NIGHTLY_FEATURE` it is automatically set to `unstable`, which is not a feature in our library. See the docs here: https://github.com/huonw/travis-cargo